test/sanity/validate-modules/main.py

```diff
@@ -1312,6 +1312,14 @@ def _validate_argument_spec(self, docs, spec, kwargs):
                 args_from_docs.add(arg)
                 args_from_docs.update(data.get('aliases', []))
 
+            for arg in args_from_argspec:
+                if not str(arg).isidentifier():
+                    self.reporter.error(
+                        path=self.object_path,
+                        code=336,
+                        msg="Argument '%s' is not a valid python identifier" % arg
+                    )
+
```

Comment:
Why is this block inside the `if docs` conditional? It doesn't seem to be dependent on `docs` at all.

Reply:
An oversight on my behalf.  I was trying to find the var I wanted, and placed the block in the wrong place.

I've moved it outside that block.